src/public/search/SearchProviderInterfaces.ts

```diff
@@ -0,0 +1,82 @@
+/*********************************************************
+ * Copyright (c) 2018 datavisyn GmbH, http://datavisyn.io
```

Comment:
@lehnerchristian Should we update the year or remove this copyright banner entirely? I checked the files and noticed a lot of inconsistencies. Beside this file there are only 2 more files with this copyright banner.

Reply:
We agreed with @dg-datavisyn that we will remove the banner for now and update all our files with a common banner later.